visualization/app/codeCharta/ui/mapTreeView/mapTreeView.level.component.html

```diff
@@ -44,7 +44,7 @@
 			<i class="flattened fa fa-eye-slash" ng-if="$ctrl.node.isFlattened" title="Flattened"></i>
 
 			<div class="small-action-button transparent">
-				<md-button class="md-fab" ng-click="$ctrl.openNodeContextMenu($event)">
+				<md-button class="md-fab" ng-click="$ctrl.openNodeContextMenu($event)" title="Open Node-Context-Menu">
```

Comment:
I am not certain where this applies to. Isn't this about expanding / collapsing folders in the directory?

Reply:
No. That button are the three dots next to a file or folder. These will open the node-context-menu as well as a right-click. We did this to let users know about this without knowing that a right-click listener exists here.

![image](https://user-images.githubusercontent.com/12533626/87314951-2ba5c480-c524-11ea-997d-ac7182e99d96.png)